src/modules/components/proposal/proposalActions/proposalActionsList/proposalActionTokenMint/proposalActionTokenMint.test.tsx

```diff
@@ -5,8 +5,23 @@ import type { IProposalActionTokenMintProps } from './proposalActionTokenMint.ap
 import { generateProposalActionTokenMint } from './proposalActionTokenMint.testUtils';
 
 jest.mock('../../../../member/memberDataListItem/memberDataListItemStructure', () => ({
-    MemberDataListItemStructure: ({ tokenAmount, tokenSymbol }: { tokenAmount: number; tokenSymbol: string }) => (
-        <div data-testid="member-data-list-item">{`${tokenAmount.toString()} ${tokenSymbol}`}</div>
+    MemberDataListItemStructure: ({
+        tokenAmount,
+        tokenSymbol,
+        href,
+    }: {
+        tokenAmount: number;
+        tokenSymbol: string;
+        href: string;
+    }) => (
+        <div data-testid="member-data-list-item">
+            <span data-testid="token-display">{`${tokenAmount.toString()} ${tokenSymbol}`}</span>
+            {href && (
+                <a data-testid="block-explorer-link" href={href}>
+                    Member block explorer link
+                </a>
+            )}
+        </div>
```

Comment:
Let's simplify the mock as:
```suggestion
    MemberDataListItemStructure: ({ tokenAmount, tokenSymbol, href }: IMemberDataListItemStructureProps) => (
        <div data-testid="member-data-list-item" data-href={href}>{`${tokenAmount.toString()} ${tokenSymbol}`}</div>
```
You can then test the proper `href` property through `screen.getByTestId('member...').dataset.href`